### DISCUSSION TITLE
Welcome to walle Discussions!

### DISCUSSION THREAD

**Discussion by jinsonli** [jinsonli/walle#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jinsonli** [comment]
这个项目的整体系统比较繁琐， 目前我做的也还不够满意。
所以不建议新手直接复刻， 特别是 linux 核心板卡、双目视觉。（履带的软胶打印有点贵，我之前是在未来工厂打印的60度软胶材质，好像是200多两条履带）
建议的是：
参考一下开源的结构源文件， 自己画一个主控PCB板。选择合适自己的主控单片机即可。
这样就可以做出适合自己的瓦力了。

**Reply by maoshihang** [reply]
求大佬用用嘉立创设计一个pcb好让我们白嫖一波
